lib/sql_string.js

```diff
@@ -82,24 +82,16 @@ SqlString.format = function(sql, values, stringifyObjects, timeZone) {
 SqlString.dateToString = function(date, timeZone) {
   var dt = new Date(date);
 
-  if (timeZone != 'local') {
-    var tz = convertTimezone(timeZone);
-
-    dt.setTime(dt.getTime() + (dt.getTimezoneOffset() * 60000));
-    if (tz !== false) {
-      dt.setTime(dt.getTime() + (tz * 60000));
+  if (timeZone !== 'local') {
+    var timezoneOffset = convertTimezone(timeZone) + dt.getTimezoneOffset();
+    if (timezoneOffset) {
+      dt.setTime(dt.getTime() + (timezoneOffset * 60000));
     }
   }
 
-  var year   = dt.getFullYear();
-  var month  = zeroPad(dt.getMonth() + 1, 2);
-  var day    = zeroPad(dt.getDate(), 2);
-  var hour   = zeroPad(dt.getHours(), 2);
-  var minute = zeroPad(dt.getMinutes(), 2);
-  var second = zeroPad(dt.getSeconds(), 2);
-  var millisecond = zeroPad(dt.getMilliseconds(), 3);
-
-  return year + '-' + month + '-' + day + ' ' + hour + ':' + minute + ':' + second + '.' + millisecond;
+  return dt.getFullYear() + '-' + zeroPad(dt.getMonth() + 1, 2) + '-' + zeroPad(dt.getDate(), 2) + ' ' +
```

Comment:
I don't agree with this format. It is a lot more cryptic than the original format.